seed/challenges/02-javascript-algorithms-and-data-structures/basic-algorithm-scripting.json

```diff
@@ -657,7 +657,6 @@
       ],
       "type": "bonfire",
       "isRequired": true,
-      "isBeta": true,
```

Comment:
I believe this challenge wasn't meant to be in beta. I'm not sure how I should confirm this.